rime_config.h.in

```diff
@@ -0,0 +1,9 @@
+#ifndef RIME_CONFIG_H_
+#define RIME_CONFIG_H_
+
+#define RIME_VERSION "1.4.0"
+
+#define IBUS_RIME_SHARED_DATA_DIR "@RIME_DATA_DIR@"
```

Comment:
Not sure if this should point to datadir of RimeData package or rime-ibus package.

They do not have to be installed under the same prefix, for example, on NixOS they will end up in:

* `/nix/store/4hq26wif0my89r1fhm7nlq2c65fniws4-brise-unstable-2017-09-16/share/rime-data/`
* `/nix/store/sgq7jraihcyvw82s8a2x6hpdr8xp2cwk-ibus-rime-1.4.0/share/rime-data/`

Reply:
Note that I did not test this pull request other than by building as I am not familiar with the input method. This should be checked by someone if the data is still correctly loaded after this change.